rollup.config.js

```diff
@@ -2,18 +2,31 @@ import path from 'path'
 import babel from '@rollup/plugin-babel'
 import resolve from '@rollup/plugin-node-resolve'
 import typescript from '@rollup/plugin-typescript'
+import esbuild from 'rollup-plugin-esbuild'
 import { sizeSnapshot } from 'rollup-plugin-size-snapshot'
 
 const createBabelConfig = require('./babel.config')
-
 const { root } = path.parse(process.cwd())
-const external = (id) => !id.startsWith('.') && !id.startsWith(root)
+const external = (id) =>
+  (!id.startsWith('.') && !id.startsWith(root)) ||
+  id.startsWith('@babel/runtime')
+
 const extensions = ['.js', '.ts', '.tsx']
 const getBabelOptions = (targets) => ({
   ...createBabelConfig({ env: (env) => env === 'build' }, targets),
   extensions,
+  comments: false,
+  babelHelpers: 'runtime',
 })
 
+function getEsbuild(target) {
+  return esbuild({
+    minify: false,
+    target,
+    tsconfig: path.resolve('./tsconfig.json'),
+  })
+}
+
 function createDeclarationConfig(input, output) {
```

Comment:
Just noticed, we have function declarations and arrow functions mixed in rollup config. (for all z/j/v)
It's not super important, but it shows some inconsistencies while reviewing. We can make all arrow functions either in this PR or in follow-up PR.